src/components/Hero/Hero.jsx

```diff
@@ -100,4 +99,4 @@ function Hero() {
 	);
 }
 
-export default Hero;
+export default Hero;
```

Comment:
no need to remove the EOF new line, follow the Prettier code formatting configuration